src/ui/structure/container.tsx

```diff
@@ -22,10 +28,18 @@ interface Props {
   testID?: string
   children: ReactNode
   style?: StyleProp<ViewStyle>
+  safeArea?: boolean
 }
 
-export const Container: React.FC<Props> = props => (
-  <SafeAreaView testID={props.testID} style={[styles.container, props.style]}>
-    {props.children}
-  </SafeAreaView>
-)
+export const Container: React.FC<Props> = props => {
+  const ContainerView = props.safeArea === false ? View : SafeAreaView
```

Comment:
Nitpick - Perhaps rename to `Wrapper`? A "Container" Component which renders a ContainerView is a mouthful :smile:

Reply:
Makes sense. That's how I named it at `yard` as well.